lib/MirrorCache/WebAPI.pm

```diff
@@ -79,6 +79,18 @@ sub startup {
 
         $rest_r->get('/folder_jobs/:id')->name('rest_folder_jobs')->to('folder_jobs#list');
         $rest_r->get('/myip')->name('rest_myip')->to('my_ip#show') if $reader;
+        
+        my $rest_user_auth = $r->any('/rest/user')->under('/')->to('session#ensure_admin')->name('ensure_admin');
+        my $rest_user_r = $rest_user_auth->any('/')->to(namespace => 'MirrorCache::WebAPI::Controller::Rest');
+        $rest_user_r->get('/')->name('rest_user')->to('table#list', table => 'Acc');
+        $rest_user_r->get('/:id')->to('table#list', table => 'Acc');
+        $rest_user_r->post('/')->to('table#create', table => 'Acc');
```

Comment:
This I'll have to remove.. we don't want to create new users, do we?

Reply:
Yes, either remove or comment it out. Maybe admins will need to create users using rest, but I don't see that such case is needed atm

Reply:
It's now commented